src/org/opendatakit/briefcase/ui/reused/FontUtils.java

```diff
@@ -22,18 +22,19 @@
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
+import java.util.Objects;
 
 public class FontUtils {
 
   public static Font getCustomFont(String name, Float size) {
 
     // if we are in a jar, then use the classloader, if not, load from filesystem
     File file = getResourceAsFile(name);
-    if (!file.exists()) {
+    if (!Objects.requireNonNull(file).exists()) {
```

Comment:
How is the NPE thrown from requireNonNull better than the NPE thrown from dereferencing a null `file`?

Reply:
This way at least it’s explicit that the value can be null.

It was suggested by IntelliJ

Reply:
Oh, I misread what it does then. Sorry.